packages/Checkboxes/src/index.tsx

```diff
@@ -114,11 +118,19 @@ const Checkboxes: FunctionComponent<ICheckboxesProps> = ({
           );
         }
       )}
+      {error && (
+        <div className="validation" data-testid="validation-message">
```

Comment:
Shouldn't this be an independent component and just import it to use it? I think the markup is repeated.